support-frontend/assets/pages/weekly-subscription-checkout/stage.jsx

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+import ProgressMessage from 'components/progressMessage/progressMessage';
+
+import { type Stage, type State } from './weeklySubscriptionCheckoutReducer';
+
+import CheckoutForm from './components/checkoutForm';
+import ReturnSection from 'pages/paper-subscription-checkout/components/thankYou/returnSection';
+import ThankYouContent from 'pages/paper-subscription-checkout/components/thankYou/thankYou';
```

Comment:
these thank you components might have commonality that needs extracting. That could be a task for the next PR

Reply:
Yup, I thought that too